### PR TITLE
bump(main/forgejo): 1.21.11-1

### PR DESCRIPTION
**Pull request by crystalcommunication**

### CODE REVIEW

File: packages/forgejo/build.sh [hunk 1]
```diff
@@ -2,12 +2,13 @@ TERMUX_PKG_HOMEPAGE=https://forgejo.org/
...
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.21.5-0"
+TERMUX_PKG_VERSION="1.21.11-1"
 TERMUX_PKG_SRCURL=https://codeberg.org/forgejo/forgejo/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=3dadabd4a80888724f24c0d8550b7770717f973e8951e5e1e04280ba3a2c500e
+TERMUX_PKG_SHA256=501a159a8ab8090aa04643f4c21473649047a3b1ecac089c316e84c2fa9a7ee6
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="dash, git"
 TERMUX_PKG_CONFFILES="etc/forgejo/app.ini"
+TERMUX_PKG_GO_USE_OLDER=true
```
Comment (Biswa96) [inline packages/forgejo/build.sh:11]: This is workaround and not a solution.

File: packages/forgejo/build.sh [hunk 2]
```diff
@@ -21,6 +21,7 @@ termux_step_make() {
 	cp -a "$TERMUX_PKG_SRCDIR" "$GOPATH"/src/forgejo.org/forgejo
 	cd "$GOPATH"/src/forgejo.org/forgejo
 
+	unset CGO_LDFLAGS
```
Comment (Biswa96) [inline packages/forgejo/build.sh:24]: Removing LDFLAGS will remove the runpath from forgejo binary. The following output is present in current forgejo version 1.21.5-0 but it is absent after this change.

```
$ readelf -d $PREFIX/bin/forgejo | grep runpath
  0x000000000000001d (RUNPATH)         Library runpath: [/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/home/builder/.termux-build/_cache/android-r26b-api-24-v1/lib]
```

**Comment by tqfx** [comment]
close #19335

**Comment by Biswa96** [comment]
I assume that the workaround is not required. The "fix" was added in scripts directory.

**Comment by tqfx** [comment]
#19335 should be closed.

**Pull request merged**